.circleci/run_browserstack_acceptance.sh

```diff
@@ -5,4 +5,14 @@ export BROWSERSTACK_BUILD_ID="${CIRCLE_BRANCH} - ${CIRCLE_BUILD_NUM}"
 COMMIT_MSG_TITLE=$(git log -n 1 --pretty=format:%s)
 export BROWSERSTACK_TEST_RUN_NAME=$COMMIT_MSG_TITLE
 
-npx testcafe "browserstack:ie@11.0,browserstack:safari" tests/acceptance/acceptancesuites/*.js -q
\ No newline at end of file
+. $(dirname $0)/testcafe_config
```

Comment:
Could we just use a testcafe config file haha. This is a cool technique though!

Reply:
https://testcafe.io/documentation/402638/reference/configuration-file

Reply:
oh. yup haha